app/components/house.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export default class House extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    if(this.props.house.imageUrl) {
+      // TODO: show the image
```

Comment:
this is just a case of returning an img tag right? Can't you do it in this PR?! ;)